doc/src/learn/tokenomics/storage-fund.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Sui’s Storage Fund
+---
+
+Sui includes an efficient and sustainable economic mechanism for financing data storage, which is important given Sui’s ability to store arbitrarily large amounts of on-chain data.
+
+Financially, on-chain data storage introduces a severe intertemporal challenge: validators who process and write data into storage today may differ from the future validators needing to store that data. If users were to pay fees for computation power only at write, effectively, future users would need to subsidize past users for their storage and pay disproportionately high fees. This negative network externality can become highly taxing for Sui in the future if left unaddressed.
+
+Sui’s economic design includes a storage fund that redistributes storage fees from past transactions to future validators. When users transact on Sui, they pay fees upfront for both computation and storage. The storage fees are deposited into a storage fund used to adjust the share of future stake rewards distributed to validators relative to SUI delegators. This design is intended to provide future Sui validators with viable business models.
+
+
+## Storage fund rewards
+
+Sui’s proof-of-stake mechanism calculates total stake as the sum of delegated stake plus the SUI tokens deposited in the storage fund. Hence, the storage fund receives a proportional share of the overall stake rewards depending on its size relative to total stake. The majority of these stake rewards –  a share $\gamma$ – are paid out to current validators to compensate for storage costs while the remaining $(1-\gamma)$ rewards are used to reinvest in the fund. In other words, stake rewards accruing to the storage fees submitted by past transactions are paid out to current validators to compensate them for data storage costs. When on-chain storage requirements are high, validators receive substantial additional rewards to compensate for their storage costs. Vice versa when storage requirements are low. 
+
+More specifically, the storage fund has three key features: 
+
+- It is funded by past transactions and functions as a tool for shifting gas fees across different epochs. This ensures that future validators are compensated for their storage costs by the past users who created those storage requirements in the first place.
+- It pays out only the returns on its capital and does not distribute its principal. That is, in practice, it is as if validators were able to borrow the storage fund’s SUI as additional stake and keep the majority of stake rewards (a $\gamma$). But note that validators do not receive funds directly from the storage fund. This guarantees the fund never loses its capitalization and can survive indefinitely. This feature is further buttressed by the $(1-\gamma)$ of stake rewards reinvested in the fund.
```

Comment:
Weirdly enough, I don't get the greeks formatted correctly here.

Reply:
> Weirdly enough, I don't get the greeks formatted correctly here.

@econmysten can you take a look? Thanks!

Reply:
> this seems to be a github problem. the math typsetting does not appear to work in bulleted lists?

I confirmed this is true by moving the lines out of the list. I tried some hacks (escaping) from this page first to no avail:
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions

Reply:
figured out a fix, leaving an empty line between the bullet points

Reply:
> figured out a fix, leaving an empty line between the bullet points

Nice. LGTM.